spec/support/have_submit_button_matcher.rb

```diff
@@ -0,0 +1,5 @@
+RSpec::Matchers.define :have_submit_button do |value|
+  match do
+    page.has_selector?("input[type=submit][value='#{value}']")
+  end
+end
```

Comment:
Could you define this as a method inside the spec where you want to use it?

```ruby
def have_submit_button(value)
  page.has_selector?("input[type=submit][value='#{value}']")
end
```

I don't think we should add new matchers if we can avoid it.

Reply:
Happy to do this 👍 - out of curiosity why should we avoid defining new matchers?

Reply:
It can lead to harder to understand tests, because if there are many, you end up reading test files while constantly looking things up to see what they're actually doing under the hood.